src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -95,9 +95,7 @@ export default function ConfirmedAppointmentListItem({
       className={itemClasses}
       data-request-id={appointment.id}
       data-is-cancelable={
-        !appointment.isCommunityCare && !appointment.vaos.videoType
-          ? 'true'
-          : 'false'
+        !appointment.isCommunityCare && !isVideo ? 'true' : 'false'
```

Comment:
Maybe this was broken before, but seems like it should be `isCommunityCare` without `appointment.`